serving-tiles/manually-building-a-tile-server-debian-11.md

```diff
@@ -72,6 +72,13 @@ While still working as the "postgres" user, set up PostGIS on the PostgreSQL dat
 
 (to exit back to be the user that we were before we did "sudo -u postgres -i" above)
 
+## Postgresql optimization
+
+If you have a default installation of postgresql it is very unlikely that it will be configured to offer good performance by default.
```

Comment:
```suggestion
If you have a default installation of PostgreSQL it is very unlikely that it will be configured to offer good performance by default.
```